bundle/internal/schema/main_test.go

```diff
@@ -0,0 +1,63 @@
+package main
+
+import (
+	"io"
+	"os"
+	"path"
+	"runtime"
+	"testing"
+
+	"github.com/stretchr/testify/assert"
+)
+
+func copyFile(src, dst string) error {
+	in, err := os.Open(src)
+	if err != nil {
+		return err
+	}
+	defer in.Close()
+
+	out, err := os.Create(dst)
+	if err != nil {
+		return err
+	}
+	defer out.Close()
+
+	_, err = io.Copy(out, in)
+	if err != nil {
+		return err
+	}
+
+	return out.Close()
+}
+
+// Checks whether descriptions are added for new config fields in the annotations.yml file
+// If this test fails either manually add descriptions to the `annotations.yml` or do the following:
+//  1. run `make schema` from the repository root to add placeholder descriptions
+//  2. replace all "PLACEHOLDER" values with the actual descriptions if possible
+//  3. run `make schema` again to regenerate the schema with acutal descriptions
+func TestRequiredAnnotationsForNewFields(t *testing.T) {
```

Comment:
Can this be an automatic hook that automatically updates the `annotations.yml` file in a PR instead of a manual step that a user has to run?

Reply:
I'd rather avoid an additional manual step we have to run everytime we add a new configurable to DABs.

Reply:
Yeah, but that could be called out during PR reviews where it'll be obvious that the description is missing and should be added.

Reply:
I think this is fine. It makes it a conscious choice to document or to not document new fields.